x-pack/plugins/security_solution/common/endpoint/models/event.ts

```diff
@@ -300,107 +350,33 @@ export function getAncestryAsArray(event: GetAncestryArrayFields | undefined): s
   return [];
 }
 
-/**
- * @param event The event to get the category for
- */
-export function primaryEventCategory(event: ResolverEvent): string | undefined {
-  if (isLegacyEvent(event)) {
-    const legacyFullType = event.endgame.event_type_full;
-    if (legacyFullType) {
-      return legacyFullType;
-    }
-  } else {
-    const eventCategories = event.event.category;
-    const category = typeof eventCategories === 'string' ? eventCategories : eventCategories[0];
-
-    return category;
-  }
-}
-
 /**
  * @param event The event to get the full ECS category for
  */
-export function allEventCategories(event: ResolverEvent): string | string[] | undefined {
-  if (isLegacyEvent(event)) {
-    const legacyFullType = event.endgame.event_type_full;
-    if (legacyFullType) {
-      return legacyFullType;
-    }
-  } else {
-    return event.event.category;
-  }
+export function eventCategory(event: SafeResolverEvent): string[] {
```

Comment:
:grey_question: Should we call it event categor_ies_ (plural)? Because it could be many or stick with categor_y_ b/c ECS?

Reply:
Not sure. Any opinion on that?

Reply:
Plural makes sense, but following the ECS paradigm is probably best imo

Reply:
Regardless of the name, the TS type indicates that it returns an array. I think we can defer this decision.